### PR TITLE
CTDC-2158: Normalize survival status text casing

### PR DESCRIPTION
**Pull request by Nahomtes**
This pull request improves the handling and display of the "Survival Status" field in the participant detail view by ensuring consistent, user-friendly formatting. A new utility function, `toTitleCase`, is introduced to convert survival status values to proper title case, and comprehensive tests are added to ensure its correctness. The controller now pre-formats the survival status value, and related tests are updated and expanded to verify the display logic.

**Survival Status Formatting and Display:**

* Added a new utility function `toTitleCase` in `src/utils/utils.js` to convert strings to proper title case, handling grammar rules and edge cases.
* Updated `ParticipantDetailController` to use `toTitleCase` for the `survival_status` field, ensuring all displayed values are consistently formatted. [[1]](diffhunk://#diff-87194a40091ae7ec5f4ef7088ccaf1e2bb040a3ba82f05a5b3cc9d5cc3f95291R12) [[2]](diffhunk://#diff-87194a40091ae7ec5f4ef7088ccaf1e2bb040a3ba82f05a5b3cc9d5cc3f95291L77-R78)

**Testing Enhancements:**

* Added comprehensive unit tests for `toTitleCase` in `src/utils/utils.test.js`, covering standard cases, complex phrases, and edge cases. [[1]](diffhunk://#diff-d3daeb7cdfad08119a4cb03d44e541535033d5cb8c84fefb40b4c5e4cfc75280L1-L14) [[2]](diffhunk://#diff-d3daeb7cdfad08119a4cb03d44e541535033d5cb8c84fefb40b4c5e4cfc75280R63-R148)
* Added and updated tests in `HeaderPanel.test.js` to verify that the survival status value is displayed as formatted by the controller, including cases with multiple-word statuses.

### CODE REVIEW

File: src/utils/utils.js [hunk 1]
```diff
@@ -35,4 +35,70 @@ export const convertCRDCLinksToValue = (data, key) => {
...
+      // Check if it's a small word
+      if (smallWords.has(word)) {
+        return word; // Keep lowercase
+      }
+
+      // Capitalize all other words
+      return word.charAt(0).toUpperCase() + word.slice(1);
+    })
+    .join(" ");
+};
```
[the remark this answers is not in the document]
Comment (Nahomtes) [reply src/utils/utils.js]: Added this suggestion

File: src/pages/participantDetail/components/HeaderPanel.test.js [hunk 2]
```diff
@@ -426,4 +426,64 @@ describe('HeaderPanel', () => {
...
+      };
+
+      // Act
+      renderComponent(participant);
+
+      // Assert
+      const survivalStatusValue = Array.from(
+        container.querySelectorAll('.infoPanelValue'),
+      )[4]; // 5th value (index 4) is Survival Status
+      expect(survivalStatusValue.textContent).toBe('Dead');
```
[the remark this answers is not in the document]
Comment (Nahomtes) [reply src/pages/participantDetail/components/HeaderPanel.test.js:465]: Added this suggestion

**Review by nathanhuangnih — APPROVED** [review]
LGTM
